src/App.vue

```diff
@@ -69,7 +69,7 @@
       </div>
     </v-fade-transition>
 
-    <SbcHeader />
+    <SbcHeader class="d-flex" />
```

Comment:
The main Sbc Header div had height 48px whereas its content had height 70px. This caused the breadcrumbs to be partly covered over by the Sbc Header. Changing the display to flex fixes this.

Reply:
This went away with the latest commit (to Vue 2.7).